### PR TITLE
[Keyboard] Add Dactyl Manuform 5x6 with 5 thumb keys

### PR DESCRIPTION
**Pull request by jceb**
## Description

Add support for the Dactyl Manuform 5x6 with 5 thumb keys

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/5x6_5/5x6_5.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    // DDRD |= (1<<5);
+    // PORTD &= ~(1<<5);
+
+    // // orange led on
+    // DDRB |= (1<<0);
+    // PORTB &= ~(1<<0);
+
+	matrix_init_user();
+};
+
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_5/5x6_5.c:23]: this block isn't needed.
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6_5/5x6_5.h [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#include "dactyl_manuform.h"
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avr/io.h>
+  #include <avr/interrupt.h>
+#endif
+#endif
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_5/5x6_5.h:12]: This block isn't needed.
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6_5/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+NKRO_ENABLE = yes            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_5/rules.mk:20]:
```suggestion
BOOTMAGIC_ENABLE = lite
COMMAND_ENABLE = no
NKRO_ENABLE = yes
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**